sources/api/migration/migrations/v1.0.8/add-bootstrap-containers/src/main.rs

```diff
@@ -0,0 +1,23 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::AddPrefixesMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added the setting `bootstrap-containers`
+fn run() -> Result<()> {
+    migrate(AddPrefixesMigration(vec![
```

Comment:
I don't think it matters, but just calling out that we are leaving the bootstrap container directories and env/userdata files behind on downgrade. That's probably fine and seems consistent with other new features we've added.